### PR TITLE
more sensible cvar defaults

### PR DESCRIPTION
**Pull request by solarisstar**

### CODE REVIEW

File: code/client/cl_parse.c [hunk 1]
```diff
@@ -429,7 +429,7 @@ void CL_SystemInfoChanged(void) {
     }
     // if game folder should not be set and it is set at the client side
```
Comment (MJLHThomassen) [inline code/client/cl_parse.c:430]: "All your base are belong to us" XD This comment is not really goodee englisjh

File: code/qcommon/vm.c [hunk 2]
```diff
@@ -64,9 +64,9 @@ VM_Init
 ==============
 */
 void VM_Init(void) {
-    Cvar_Get("vm_cgame", "2", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 2
-    Cvar_Get("vm_game", "2", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 2
-    Cvar_Get("vm_ui", "2", CVAR_ARCHIVE);		// !@# SHIP WITH SET TO 2
+    Cvar_Get("vm_cgame", "0", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 0
```
Comment (MJLHThomassen) [inline code/qcommon/vm.c:74]: Perhaps make a comment here to indicate what the different values mean, or is this already commented somewhere where vm_ui etc is defined in one of those large structs? (if it is defined in a struct that is).

Comment (solarisstar) [inline code/qcommon/vm.c:74]: Good idea.

File: code/client/cl_parse.c (hunk 1 above)
Comment (solarisstar) [inline code/client/cl_parse.c:430]: But I have such a good time reading these comments sometimes.

"lol xD"

**Pull request merged**